alfred/files/alfred.procd

```diff
@@ -103,41 +94,38 @@ alfred_start()
 	return 0
 }
 
-start()
-{
+start_service() {
 	config_load "alfred"
 	config_foreach alfred_start alfred
-
-	if [ "$enable" = "0" ]; then
-		exit 0
-	fi
-
-	echo "${initscript}: starting alfred"
-	SERVICE_PID_FILE="$pid_file_alfred"
-	service_start /usr/sbin/alfred ${alfred_args}
-
-	if [ "$vis_enable" = "1" ]; then
-		echo "${initscript}: starting batadv-vis"
-		SERVICE_PID_FILE="$pid_file_vis"
-		service_start /usr/sbin/batadv-vis ${vis_args}
-	fi
-
-	if [ "$run_facters" = "1" ]; then
+	procd_open_instance "alfred"
+	echo "starting alfred"
```

Comment:
Why this echo? Other procd scripts also don't do this.

Reply:
The echo was there [before](https://github.com/openwrt-routing/packages/pull/390/files#diff-9cac72d591240485e6d7dd48f4aaa90eL115) and I though of sticking as much to the "original" init file as possible